book/src/SUMMARY.md

```diff
@@ -0,0 +1,18 @@
+# Summary
+
+- [Introduction](introduction.md)
+- [Tutorial](tutorial/introduction.md)
+  - [Creating a UEFI Application](tutorial/app.md)
```

Comment:
as I'm not a native speaker, I often mix up `a/an`. However, here, it feels like it should be `an UEFI Application`. What do you think?

Reply:
"a" is correct here. The rule is to check if the following word is pronounced out loud with a consonant, regardless of whether it literally is, and use "a" if so. So for example it's "a university" rather than "an university", because the "u" here is pronounced like "yoo". Same thing with "UEFI".

Words that start with "u" and "h" are often edge cases in this regard, and so even among native English speakers you will see some mixups here. In fact the UEFI spec is a good example in that they usually use "a UEFI" (dozens of times), but in about 20 cases they use "an UEFI" :laughing:

Reply:
Haha, thanks for the explanation. I guess, this is as unintuitive, as German articles before nouns are for English speakers 😅